src/components/AvatarTypes.ts

```diff
@@ -0,0 +1,13 @@
+import CONST from '@src/CONST';
+
+type AvatarFunctionType = () => void;
+
+type AvatarType = {
```

Comment:
I'm not sure about this name. Looks like You use it in an icons attribute, so maybe AvatarIcon is more accurate? Also, the file name and variable are not consistent 'AvatarTypes' vs 'AvatarType'

Reply:
renamed